parade/ddl.sql

```diff
@@ -48,6 +50,7 @@ $$;
 CREATE OR REPLACE FUNCTION own_tasks(
     max_tasks INTEGER, actions VARCHAR(128) ARRAY, owner_id VARCHAR(64), max_duration INTERVAL
 )
+-- tzahi: you may gain performance by partitioning on status
```

Comment:
#780

Reply:
It certainly depend on where your performance issues come from.
You would need to define you primary key an status + ID. 
It should speed up Own task. As owned rows will be moved to the 'in-progress' partition, pending partition will contaning only 'pending' tasks. so the scan should be very fast. 
When updating the status, rows will move to another partition. I do not know if and how much it is slower than simple update.
the 'in-progress' partition will be relatively small, so scanning it will be fast anyway.

Reply:
I think it will speed own_task immensely